<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIClient</class>
 <widget class="QDialog" name="UIClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UIClient</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>270</width>
     <height>339</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Client</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Server IP:</string>
        </property>
        <property name="buddy">
         <cstring>ServAddrEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ServAddrEdit">
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Server Port:</string>
        </property>
        <property name="buddy">
         <cstring>servPortEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="servPortEdit">
        <property name="text">
         <string>8888</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>81</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>81</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Choose File:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fileNameEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="fileChooseBnt">
        <property name="maximumSize">
         <size>
          <width>27</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>srv  addr:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fileServAddr">
        <property name="text">
         <string>/home/baiyun/testTran/</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="transSpeed">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fileSendBnt">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="statusInfo">
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>140</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextEditable|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>ServAddrEdit</tabstop>
  <tabstop>servPortEdit</tabstop>
  <tabstop>fileSendBnt</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>2</number>
  </property>
  <property name="gridDeltaY">
   <number>2</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
